### DISCUSSION TITLE
Immediately requesting the pronunciation if there's no audio

### DISCUSSION THREAD

**Discussion by it0na72** [realmayus/anki_forvo_dl#24] [Ideas]
Thanks for the amazing addon you made. Works incredibly well.
I was just wondering if it would even be possible to add the request of the word pronunciation immediately if there's no pronunciation done? So if I try to add the pronunciation of a word in anki and it can't find it, it would send a request to the website and request the pronunciation. 
Not sure if it is possible but would be an amazing feature to implement. 

Thanks in advance.

**Comment by realmayus** [comment]
Hi! This feature would unfortunately require a lot of rework to be done, since the plugin doesn't use any user accounts right now, which are required for adding requests. 
I don't have any plans for implementing this as of now, but i'll keep this post 👍
